modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/folders/SuggestedFoldersPaywallBottomSheet.kt

```diff
@@ -0,0 +1,62 @@
+package au.com.shiftyjelly.pocketcasts.podcasts.view.folders
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.FrameLayout
+import androidx.core.view.doOnLayout
+import au.com.shiftyjelly.pocketcasts.compose.AppTheme
+import au.com.shiftyjelly.pocketcasts.compose.extensions.contentWithoutConsumedInsets
+import au.com.shiftyjelly.pocketcasts.settings.onboarding.OnboardingFlow
+import au.com.shiftyjelly.pocketcasts.settings.onboarding.OnboardingLauncher
+import au.com.shiftyjelly.pocketcasts.settings.onboarding.OnboardingUpgradeSource
+import au.com.shiftyjelly.pocketcasts.ui.theme.Theme
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.google.android.material.bottomsheet.BottomSheetDialog
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import dagger.hilt.android.AndroidEntryPoint
+import javax.inject.Inject
+
+@AndroidEntryPoint
+class SuggestedFoldersPaywallBottomSheet : BottomSheetDialogFragment() {
+
+    @Inject
+    lateinit var theme: Theme
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ) = contentWithoutConsumedInsets {
+        AppTheme(theme.activeTheme) {
+            SuggestedFoldersPaywall(
```

Comment:
We should handle window insets and not draw the content behind the camera punch hole.

![410124626-120bd39b-bafd-42b5-8f17-0be36c14a7c9](https://github.com/user-attachments/assets/6f3d50dd-ee96-41b0-9888-f4b8bddaea03)

Reply:
[8bd1b92](https://github.com/Automattic/pocket-casts-android/pull/3554/commits/8bd1b924c992f988d0f234d27098a8703ea41415)